util/notification.py

```diff
@@ -105,3 +123,13 @@ def get_notification_data(notification_type: str) -> Dict[str, str]:
             f"No data for type {notification_type} found."
         )
     return notification_data
+
+
+def __check_endpoint_set():
+    endpoint = os.getenv("WS_NOTIFY_ENDPOINT")
```

Comment:
we have a const reference for this WEBSOCKET_ENDPOINT. Though I don't know if env variables might change during runtime (but we never checked that on other occasions so i'd go with the const)

Reply:
Dont exactly know what you mean so maybe lets talk about this

Reply:
we don't need to get the os env variable with every call of the function. In the notification.py file is a const reference to the endpoint 
![image](https://user-images.githubusercontent.com/91723236/222372018-5be25d6d-345a-44c3-8162-ec56999c55c0.png)

Reply:
Ah ok, yeah this makes sense